.travis.yml

```diff
@@ -7,9 +7,8 @@ branches:
   only:
     - gh-pages
 
-rvm:
-  - 2.2.3
-script: bundle exec jekyll build && htmlproof --check-html --disable-external ./_site
+rvm: 2.2.3
+script: bundle exec jekyll build && bundle exec htmlproofer --check-html --disable-external ./_site --file-ignore ./_site/index.html
```

Comment:
Because I don't get the error on any off my machines, linux or mac after testing for a whole weekend. I couldn't find it. Even online checker say, this file is okay, so i ignore a error on travis I can't reproduce or understand.